addon/-private/system/normalize-link.ts

```diff
@@ -10,7 +18,7 @@
   @return {Object|null}
```

Comment:
the documentation says that link should be a string, but the code says it can be an object. 
She these docs be updated?

Reply:
we need a subject matter expert to comment on this. @runspired @igorT

Reply:
The docs should be updated